doc_src/history.txt

```diff
@@ -44,4 +46,9 @@ history --search --contains "foo"
 
 history --delete --prefix "foo"
 # Interactively deletes the record of previous commands which start with "foo".
+
+\subsection history-notes Notes
+
+If you specify both `--prefix` and `--contains` the last flag seen is used.
```

Comment:
Sorry, I read the code wrong - I thought `history --delete --search` would search, not delete. Seems that prints an error as it should.